content/show/linux-action-news/_index.md

```diff
@@ -15,14 +15,20 @@ header_image = "/images/shows/linux-action-news.png"
 support_link = "https://patreon.com/jupitersignal"
 
 podverse_podcast_id = "haAiVbD1N9"
-+++
 
-Weekly Linux news and analysis by hosts Chris and Joe. We hope this is the show you’ll go to each week when you want to hear an informed discussion about what’s happening in Linux.
 
-<ul>
-<li><a href="https://linuxactionnews.com/hosts/joe">Joe Ressington</a></li>
-<li><a href="https://linuxactionnews.com/hosts/chris">Chris Fisher</a></li>
-<li><a href="https://linuxactionnews.com">Show Notes</a></li>
-<li><a href="https://www.jupiterbroadcasting.com/contact/">E-Mail</a></li>
-<li><a href="https://twitter.com/linuxactionnews">Twitter</a></li>
-</ul>
+[links.twitter]
+  url = "https://twitter.com/linuxactionnews"
+[links.email]
+  url = "https://www.jupiterbroadcasting.com/contact/"  
+[links.reddit]
+  url = "https://www.reddit.com/r/coderradio"
```

Comment:
coder radio reddit eh?
Hmm... not seeing this one on LAN.com.. 
Could you remove this entry?

Reply:
Good catch. I updated it.